### DISCUSSION TITLE
Deskripsi Transaksi: Fomat Data dibuat konsisten

### DISCUSSION THREAD

**Discussion by 3xploiton3** [buku-masjid/buku-masjid#77] [General]
![Daftar_Transaksi_-_Buku_Masjid_Admin_-_Google_Chro_27-05-2024-(-)](https://github.com/buku-masjid/buku-masjid/assets/19517680/8d5b4d40-9f1c-4a03-b076-8d7da5a04a07)

Bagaimana pendapat kalian jika format kolom menulis, data yang tampil harus persis sama, agar format konsisten, 

ini berlaku untuk setiap format inputan data, jikat tidak dibolehkan ada enter (baris baru) kolom isi dibuat 1 baris bukan _long text_

**Comment by nafiesl** [comment]
Halo mas @3xploiton3 terima kasih informasinya. Baik bisa jadi issue untuk diperbaiki. Sebagai solusi cepatnya, mas bisa coba update kodenya seperti ini:

```diff
- {{ $transaction->description }}
+ {!! nl2br(htmlentities($transaction->description)) !!}
```
Tepatnya dibagian sini:
https://github.com/buku-masjid/buku-masjid/blob/8b8dc909facc9fe7191477e83d1d9085de6a5a08/resources/views/transactions/index.blade.php#L71


Berikut demonya.

https://github.com/buku-masjid/buku-masjid/assets/8721551/53e11b2d-9406-493d-ac83-13d1665cd178

**Reply by masjidalhidayah** [reply]
pak yang untuk [masjid al hidayah](https://alhidayah-mondokan.bukumasjid.com/) / semuanya, bisa di ubah seperti ini juga ?
lebih nyaman dalam pembacaannya, terimakasih


![image](https://github.com/user-attachments/assets/faf4c4be-c954-4ac7-af7c-b69f77792439)
![image](https://github.com/user-attachments/assets/139617d5-b780-4c31-9256-b9dd3a91a1a1)

**Comment by nafiesl** [comment]
Halo mas @3xploiton3 dan @masjidalhidayah masukan ini sudah kita akomodir pada commit 85ccae1b7f03202523229916e7a7d4b82ab112ca

Tampilan pada list transaksi (admin)
![screen_2025-06-11_002](https://github.com/user-attachments/assets/46ba637c-95a4-47cd-9c96-cc2792a7cd5d)

Tampilan pada laporan per kategori (halaman admin)
![screen_2025-06-11_001](https://github.com/user-attachments/assets/fae88593-bf4f-4435-a645-63daa3965084)

Tampilan PDF laporan per kategori (halaman admin)
![screen_2025-06-11_003](https://github.com/user-attachments/assets/4ac4e069-51d8-4817-97a0-4936ac3ff549)

Tampilan laporan publik
![screen_2025-06-11_004](https://github.com/user-attachments/assets/94a6274a-f471-4905-9b6a-73554fc0cdbd)

Dengan demikian, diskusi ini kita tutup. Terima kasih mas2.
